app/backend/gunicorn.conf.py

```diff
@@ -0,0 +1,10 @@
+import multiprocessing
+
+max_requests = 1000
+max_requests_jitter = 50
+log_file = "-"
+bind = "0.0.0.0"
+
+workers = (multiprocessing.cpu_count() * 2) + 1
+threads = workers
```

Comment:
the threads argument is the number of threads per worker, so it should be 1 or 2. 

I've found 1 thread is best when there is only 1 CPU core (F1, B1, S1, P1V2) and 2 when there are 2 or more.

Reply:
Oh interesting, I misinterpreted the docs due to its suggestion of 2-4 x cores: https://docs.gunicorn.org/en/latest/settings.html#threads
I've updated per your suggestion.